modules/dxp/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/client/AsahFaroBackendClientImpl.java

```diff
@@ -370,6 +352,61 @@ private MultivaluedMap<String, Object> _getParameters(
 		return uriVariables;
 	}
 
+	private String _getResponse(
+			long companyId, Http.Method method,
+			MultivaluedMap<String, Object> parameters, String path,
```

Comment:
Why do we need to use a MultivaluedMap?

Reply:
Because we have a Multivalued parameters like sort parameter that we are already using, so, for that reason we are using it. Of corse that, we could use another implementation but I think that this is a good implementation and fit good to perform this need.